code/modules/reagents/chemistry/reagents/water.dm

```diff
@@ -251,6 +251,7 @@
 		if(isvampirethrall(M))
 			M.mind.remove_antag_datum(/datum/antagonist/mindslave/thrall)
 			holder.remove_reagent(id, volume)
+			M.visible_message("<span class='danger'>[M] recoils, their skin flushes with color, regaining their sense of control!</span>")
```

Comment:
Don't we use english spelling for strings but american spelling for actual "code"?

*I legit aint sure, but I know the wiki is in proper english, not yank*